packages/block-editor/src/components/link-control/index.js

```diff
@@ -269,7 +269,6 @@ function LinkControl( {
 
 		return Promise.resolve( [
 			{
-				id: val,
```

Comment:
I'm not totally clear on what this change does, any background on the need to remove this?

Reply:
The manual entries have no `id` anymore so I removed it from this resolve.